<launch>

   <node name="electro_optical" pkg="eeyore_ros" type="electro_optical_node" output="screen" />
   <rosparam file="$(find eeyoreROS)/config/electro_optical.yaml"/>

</launch>
